packages/cosmos_ui_components/lib/components/cosmos_bottom_sheet_container.dart

```diff
@@ -26,3 +26,21 @@ class CosmosBottomSheetContainer extends StatelessWidget {
     );
   }
 }
+
+Future showCosmosBottomSheet(Widget child, BuildContext context, {bool isFullScreen = true}) => showModalBottomSheet(
+      context: context,
+      isScrollControlled: true,
+      enableDrag: true,
+      shape: RoundedRectangleBorder(
+        borderRadius: BorderRadius.vertical(
+          top: Radius.elliptical(
+            CosmosTheme.of(context).radiusL,
+            CosmosTheme.of(context).radiusL * 1.1,
+          ),
+        ),
+      ),
+      builder: (context) => SizedBox(
+        height: isFullScreen ? MediaQuery.of(context).size.height / 1.06 : MediaQuery.of(context).size.height / 2,
+        child: CosmosBottomSheetContainer(child: child),
+      ),
+    );
```

Comment:
you can reuse the code from here to show the modal bottom sheet:
https://github.com/tendermint/flutter/blob/c8cc456dfdd95539ebccea0e632df9105a3035b7/starport_template/lib/pages/create_wallet_page.dart#L117-L123

its in the #129 pr.

Reply:
This `showMaterialModalBottomSheet` has some extra odd implicit spacing from the top which looks very weird. Using it for now but we will either have to figure out to use something else, or build our own. I am not removing the current one (we might use that if we decide). Putting a TODO

![Screenshot 2021-11-24 at 10 46 03 PM](https://user-images.githubusercontent.com/25613623/143289229-4068d9fc-7397-43b7-afda-4b9978130adc.png)